tests/util.go

```diff
@@ -93,6 +93,29 @@ func RemoveNamespaces() {
 	}
 }
 
+func CreateNamespaceWithParameters(namespace string) error {
+	output, stderr, err := ktests.RunCommandWithNS("", "oc", "new-project", namespace)
+	if err != nil {
+		if strings.Contains(stderr, fmt.Sprintf("Error from server (AlreadyExists): project.project.openshift.io \"%s\" already exists", namespace)) {
+			err = nil
+		} else {
+			err = fmt.Errorf("Create project: command os new-project %s: output %s: stderr %s: %v", namespace, output, stderr, err)
+		}
+	}
+	return err
+}
+func RemoveNamespaceWithParameters(namespace string) error {
+	output, stderr, err := ktests.RunCommandWithNS("", "oc", "delete", "project", namespace)
+	if err != nil {
+		if strings.Contains(stderr, fmt.Sprintf("Error from server (NotFound): namespaces \"%s\" not found", namespace)) {
+			err = nil
+		} else {
+			err = fmt.Errorf("Delete project: command os delete project %s: output %s: stderr %s: %v", namespace, output, stderr, err)
```

Comment:
Error messages in Golang idiomatically start with a lowercase letter.

Reply:
I have updated the Error messages starting with a lowercase letter